.changes/next-release/Feature-666cb5e9-eff1-41d9-bcf8-1d65cda2dd2c.json

```diff
@@ -0,0 +1,4 @@
+{
+	"type": "Feature",
+	"description": "Use MDE endpoint set by environment variable"
```

Comment:
Thanks for including a changelog, but in this case it's not really customer-relevant so we should probably omit it.